apps/coscrad-frontend/src/utils/generic-components/presenters/id-info-icon.tsx

```diff
@@ -1,16 +1,77 @@
 import { ResourceType } from '@coscrad/api-interfaces';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
 import InfoIcon from '@mui/icons-material/Info';
-import { Box, Tooltip } from '@mui/material';
+import {
+    Dialog,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+    IconButton,
+    Tooltip,
+    Typography,
+} from '@mui/material';
+import { useState } from 'react';
 
 interface IdInfoIconProps {
     id: string;
     type: ResourceType;
 }
 
-export const IdInfoIcon = ({ id, type }: IdInfoIconProps): JSX.Element => (
-    <Box component="span">
-        <Tooltip title={`ID: ${type}/${id}`}>
-            <InfoIcon />
-        </Tooltip>
-    </Box>
-);
+export const IdInfoIcon = ({ id, type }: IdInfoIconProps): JSX.Element => {
```

Comment:
Is it possible to write a Cypress test of this for just one particular resource type?

Reply:
I can test if the button to copy is visible, but to access the clipboard the user has to allow the cypress browser access to the clipboard.  It wasn't able to log the result I was looking for from the clipboard, but also I assume the CI/CD system won't be able to make access to the clipboard possible.

Should I just leave it at checking the copy icon is present?

Reply:
No. Cypress has access to the window, location, navigation, etc. Let me know if you need help.
I haven't read it closely, but [this](https://dev.to/walmyrlimaesilv/testing-copy-to-clipboard-with-cypress-1414) looks like it may be useful.